src/indieweb_utils/indieauth/server.py

```diff
@@ -75,6 +75,36 @@ def validate_authorization_response(
     :type allowed_methods: list
     :returns: A boolean indicating whether the response is valid.
     :rtype: bool
+
+    Example:
+
+    .. code-block:: python
+
+        import indieweb_utils
+
+        grant_type = "authorization_code"
+        code = "12345"
+        client_id = "https://example.com"
+        redirect_uri = "https://example.com/callback"
+        code_challenge = "12345"
+        code_challenge_method = "S256"
+        allowed_methods = ["S256"]
+
+        # validate authorization response
+        # if response is invalid, an exception will be raised
+        try:
+            indieweb_utils.validate_authorization_response(
+                grant_type,
+                code,
+                client_id,
+                redirect_uri,
+                code_challenge,
+                code_challenge_method,
+                allowed_methods
+            )
+        except Exception as e:
```

Comment:
Can we catch a specific error for this rather than the overly broad Exception? It would also save us a code comment in our docs.